packages/atlas/src/components/Fee/Fee.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from '@emotion/styled'
+import { FC } from 'react'
+
+import { Information } from '@/components/Information'
+import { NumberFormat } from '@/components/NumberFormat'
+import { Text, TextVariant } from '@/components/Text'
+import { Color } from '@/components/Text/Text.styles'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+
+export type FeeProps = {
+  amount: number
+  variant: TextVariant
+  color?: Color
+  hideOnMobile?: boolean
+  className?: string
+  loading?: boolean
+}
+
+export const Fee: FC<FeeProps> = ({
+  amount,
+  variant = 't100',
+  color = 'colorTextStrong',
+  hideOnMobile,
+  className,
+  loading,
+}) => {
+  const smMatch = useMediaMatch('sm')
+  return (
+    <Wrapper className={className}>
+      {(!hideOnMobile || smMatch) && (
+        <>
+          <Text as="span" variant={variant} color={color}>
+            Fee:&nbsp;{amount < 0.01 && '<'}&nbsp;
+          </Text>
+          <NumberFormat
+            displayedValue={amount < 0.01 ? 0.01 : undefined}
+            value={amount}
+            as="span"
+            variant={variant}
+            color={loading ? 'colorText' : color}
```

Comment:
are you sure that's `colorText`? I think this is supposed to be `Text/Default/Muted`